src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import React from 'react';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map(tab => (
+              <li
+                className={selectedTab === tab ? 'is-active' : null}
```

Comment:
[CHECKLIST P.2](https://github.com/mate-academy/react_tabs-js/blob/master/checklist.md) WE suggest you never compare the object with another object with strict comparison. It could produce a lot of bugs. Compare object using their unique identifier (id, title)

Also, it's good to use classnames library for classes manipulation